src/main/java/com/evokly/kafka/connect/mqtt/MqttSourceConnectorConfig.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2016 Evokly S.A.
+ * See LICENSE file for License
+ **/
+
+package com.evokly.kafka.connect.mqtt;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * MqttSourceConnectorConfig is responsible for correct configuration management.
+ */
+public class MqttSourceConnectorConfig {
+    private static final Logger log = LoggerFactory.getLogger(MqttSourceConnector.class);
+
+    List<Map<String, String>> mConfigs = new ArrayList<>();
+    Map<String, String> mProperties;
+
+    /**
+     * Transform process properties.
+     *
+     * @param properties associative array with properties to be process
+     */
+    public MqttSourceConnectorConfig(Map<String, String> properties) {
+        log.info("Initialize transform process properties");
+
+        mProperties = properties;
+
+        Integer index = 0;
+        Integer size = Integer.valueOf(properties.get(MqttSourceConstant.CONNECTIONS));
+
+        for (int i = 0; i < size; i++) {
+            Map<String, String> config = new HashMap<>();
+            mConfigs.add(i, config);
+
+            processProperty(i, MqttSourceConstant.KAFKA_TOPIC);
+            processProperty(i, MqttSourceConstant.MQTT_CLIENT_ID);
+            processProperty(i, MqttSourceConstant.MQTT_CLEAN_SESSION);
+            processProperty(i, MqttSourceConstant.MQTT_CONNECTION_TIMEOUT);
+            processProperty(i, MqttSourceConstant.MQTT_KEEP_ALIVE_INTERVAL);
+            processProperty(i, MqttSourceConstant.MQTT_SERVER_URIS);
+            processProperty(i, MqttSourceConstant.MQTT_TOPIC);
+            processProperty(i, MqttSourceConstant.MQTT_QUALITY_OF_SERVICE);
+        }
+
+        for (Map<String, String> configs : mConfigs) {
```

Comment:
maybe wrap with `if log.isDebugEnabled()`

and also... think that .debug is more appropriate